mantidproject/mantid
# Add boxes to DGSPlanner layout

**Description of work.**

Widgets were rearranged into group of items based on their logical function.

**Report to:** Matt Stone, @AndreiSavici 

**To test:**



Open DGSPlanner in Mantid Workbench and confirm that the layout on the left-hand side contains 3 boxes that match the image below

![image](https://user-images.githubusercontent.com/5920552/94598296-bf05b080-025c-11eb-8f0a-91e4688c6653.png)

Fixes https://code.ornl.gov/sns-hfir-scse/spectroscopy/spectroscopy/-/issues/23

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
